modules/dfpAdServerVideo.js

```diff
@@ -249,6 +254,74 @@ function getCustParams(bid, options, urlCustParams) {
   return encodedParams;
 }
 
+export async function getVastForLocallyCachedBid(gamVastWrapper, bid) {
+  const blobUrl = vastsLocalCache.get(bid.videoCacheKey);
+  if (!blobUrl) return gamVastWrapper;
+  const finalVast = await replaceVastAdTagWithBlobContent(gamVastWrapper, blobUrl, bid.videoCacheKey);
+  return finalVast;
+};
+
+async function getVast(options) {
+  const { bid } = options;
+  const vastUrl = buildDfpVideoUrl(options);
+  const response = await fetch(vastUrl);
```

Comment:
I would advise against this; the IMA SDK should be requesting GAM URLs. THe IMA SDK appends query params so my understanding is you would get different responses. Unwrapping the VAST should be opt-in